redaxo/bin/console

```diff
@@ -8,4 +8,28 @@ $REX['BACKEND_FOLDER'] = 'redaxo';
 
 chdir(dirname(__DIR__));
 
+// Check if posix is available
```

Comment:
Diese Datei wird nie geupdatet!
Wenn müsste es daher eher in `redaxo/src/core/console.php`.

Reply:
ok.